src/components/GoogleAddressLookup/index.js

```diff
@@ -1,18 +1,21 @@
-import React from 'react';
+import React, { useCallback } from 'react';
 import PropTypes from 'prop-types';
 import scriptLoader from 'react-async-script-loader';
 import PlacesLookupComponent from './component';
 import * as CustomPropTypes from './proptypes';
 
+const googleMapApiUrl = 'https://maps.googleapis.com/maps/api/js';
+
 /**
  * The GoogleAddressLookup component is used to find a location.
  * @category Form
  */
 export default function GoogleAddressLookup(props) {
     const { apiKey, ...rest } = props;
 
-    const Component = scriptLoader(
-        `https://maps.googleapis.com/maps/api/js?key=${apiKey}&libraries=places`,
+    const Component = useCallback(
+        scriptLoader(`${googleMapApiUrl}?key=${apiKey}&libraries=places`),
+        [apiKey],
     )(PlacesLookupComponent);
```

Comment:
let's use like in the map components
```suggestion
    const Component = useCallback(
        scriptLoader(`${googleMapApiUrl}?key=${apiKey}&libraries=places`)(PlacesLookupComponent),
        [apiKey],
    );
```